src/actions/actionProperties.tsx

```diff
@@ -10,35 +10,35 @@ import {
   ArrowheadArrowIcon,
   ArrowheadBarIcon,
   ArrowheadDotIcon,
-  ArrowheadTriangleIcon,
   ArrowheadNoneIcon,
-  StrokeStyleDashedIcon,
-  StrokeStyleDottedIcon,
-  TextAlignTopIcon,
-  TextAlignBottomIcon,
-  TextAlignMiddleIcon,
-  FillHachureIcon,
+  ArrowheadTriangleIcon,
+  EdgeRoundIcon,
+  EdgeSharpIcon,
   FillCrossHatchIcon,
+  FillHachureIcon,
   FillSolidIcon,
+  FillZigZagIcon,
+  FontFamilyCodeIcon,
+  FontFamilyNormalIcon,
+  FontSizeExtraLargeIcon,
+  FontSizeLargeIcon,
+  FontSizeMediumIcon,
+  FontSizeSmallIcon,
+  FreedrawIcon,
   SloppinessArchitectIcon,
   SloppinessArtistIcon,
   SloppinessCartoonistIcon,
+  StrokeStyleDashedIcon,
+  StrokeStyleDottedIcon,
   StrokeWidthBaseIcon,
   StrokeWidthBoldIcon,
   StrokeWidthExtraBoldIcon,
-  FontSizeSmallIcon,
-  FontSizeMediumIcon,
-  FontSizeLargeIcon,
-  FontSizeExtraLargeIcon,
-  EdgeSharpIcon,
-  EdgeRoundIcon,
-  FreedrawIcon,
-  FontFamilyNormalIcon,
-  FontFamilyCodeIcon,
-  TextAlignLeftIcon,
+  TextAlignBottomIcon,
   TextAlignCenterIcon,
+  TextAlignLeftIcon,
+  TextAlignMiddleIcon,
   TextAlignRightIcon,
-  FillZigZagIcon,
+  TextAlignTopIcon,
```

Comment:
sorry for the big diff here, my VS Code extension [JS/TS Import/Export Sorter](https://marketplace.visualstudio.com/items?itemName=dozerg.tsimportsorter) sorted these

if this is an unwanted change, I can revert this part

Reply:
yeah, I would revert the sorting, and potentially leave it for a separate PR and adding a lint rule for it (but that's an open question).

Reply:
also fixed in #6550